views/newsletter/templates/blocks/social/settings.hbs

```diff
@@ -1,5 +1,29 @@
 <h3><%= __('Select icons') %></h3>
-<div id="mailpoet_social_icons_selection"></div>
+<div class="mailpoet_form_field">
+    <div class="mailpoet_form_field_title"><%= __('Alignment') %></div>
+    <div class="mailpoet_form_field_radio_option">
+        <label>
+            <input type="radio" name="alignment" class="mailpoet_social_block_alignment" value="left" {{#ifCond model.styles.block.textAlign '===' 'left'}}CHECKED{{/ifCond}}/>
+            <%= __('Left') %>
```

Comment:
Would some translation context here be helpful? Without any context, it may be hard for the translator to choose the right word. For example in Czech, what would you use? Levy, Leva, Leve, Vlevo?

The same for the other translations: Center and Right.